.gitignore

```diff
@@ -4,7 +4,7 @@ pom.xml.versionsBackup
 TESTS-TestSuites.xml
 *.db
 */dat/out
-
+/.settings/
```

Comment:
You are excluding this folder, however, the repositoriy has the folder with files, and the modules too